UnityProject/Assets/Scripts/Core/Lifecycle/PlayerSpawn.cs

```diff
@@ -44,6 +60,19 @@ public interface IOnPlayerLeaveBody
 	public void OnPlayerLeaveBody(PlayerInfo account);
 }
 
+/// <summary>
+/// This interface will be called when a player is transferred out of their body
+/// </summary>
+public interface IOnPlayerLosePossess
+{
+	/// <summary>
+	/// Called on server when the player leaves a body (interface called on the old player object)
+	/// </summary>
+	/// <param name="account">The mind of the player leaving the body</param>
+	public void OnPlayerLosePossess(Mind Mind);
+}
+
+
 
 public interface IClientPlayerLeaveBody
```

Comment:
so what's the difference between losing possession and leaving the body?

Reply:
renamed

Reply:
leaving the body is  OnPlayerLeaveBody, 
losing possession this is when you remove access to view slots / that type of thing

nothing uses it yet though